02.general/19.dns-filtering-syntax/docs.en.md

```diff
@@ -18,14 +18,21 @@ DNS filtering supports two types of rules:
     * Rules with other modifiers will be ignored
 
 * “Hosts” rules, which is basically the same as `/etc/hosts`
+* Domains rules, which are simply domains
 
 Examples:
 
-1. `||example.org^`- block access to the example.org domain and all it's subdomains
-2. `@@||example.org^` - unblock access to the example.org domain and all it's subdomains
-3. `0.0.0.0 example.org` - AdGuard will now return 0.0.0.0 address for the example.org domain (but not it's subdomains)
+1. `||example.org^`- block access to the example.org domain and all its subdomains
+2. `@@||example.org^` - unblock access to the example.org domain and all its subdomains
+3. `0.0.0.0 example.org` - AdGuard will now return 0.0.0.0 address for the example.org domain (but not its subdomains)
+4. `example.org` - treated as equivalent to #3
 4. `! Here goes a comment` - just a comment
 5. `# Also a comment` - just a comment
 6. `/REGEX/` - block access to the domains matching the specified regular expression
 
-Use this syntax to create or import rules to DNS filter. If you have any questions, feel free to post them on our [forum](https://forum.adguard.com/index.php?forums/69/).
\ No newline at end of file
+Advanced examples:
+
+7. `||example.*^` - wildcard asterisks can be used in domains (Also works with simply `example.*`)
+8. `93.184.216.34 example.org` - AdGuard is also able to return non-localhost addresses for domains
+
+Use this syntax to create or import rules to a DNS filter. If you have any questions, feel free to post them on our [forum](https://forum.adguard.com/index.php?forums/69/).
```

Comment:
Based on the examples above, you can create rules for DNS filters. If you have any questions, feel free to post them on our [forum](https://forum.adguard.com/index.php?forums/69/).

Reply:
Thanks for the suggestion. I seem to have implemented it now.